### PR TITLE
[Load Balancing] Add Limitations to Dev Docs

### PR DESCRIPTION
**Pull request by patriciasantaana**
PCX-2028

### CODE REVIEW

File: content/load-balancing/understand-basics/limitations.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+---
+pcx-content-type: concept
```
Comment (kodster28) [inline content/load-balancing/understand-basics/limitations.md]:
```suggestion
pcx-content-type: reference
```

File: content/load-balancing/understand-basics/limitations.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+title: Limitations
+weight: 18
+---
+# Limitations
```
Comment (kodster28) [inline content/load-balancing/understand-basics/limitations.md:7]:
```suggestion

# Limitations
```

File: content/load-balancing/understand-basics/limitations.md [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+| Name | Non-Enterprise | Enterprise | Maximum/Minimum Value for Enterprise |
```
Comment (kodster28) [inline content/load-balancing/understand-basics/limitations.md]:
```suggestion
| Name | Non-Enterprise | Enterprise | 
```

File: content/load-balancing/understand-basics/limitations.md [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+| --- | --- | --- | --- |
+| Load balancers | 20 | 20 | 100 |
```
Comment (kodster28) [inline content/load-balancing/understand-basics/limitations.md]:
```suggestion
| Load balancers | 20 | 100 |
```

File: content/load-balancing/understand-basics/limitations.md [hunk 5]
```diff
@@ -0,0 +1,13 @@
...
+| Monitor intervals | 60, 3600 | 10, 3600 | 10, 3600 |
```
Comment (kodster28) [inline content/load-balancing/understand-basics/limitations.md]:
```suggestion
| Monitor intervals | 60 (min), 3600 (max) | 10 (min), 3600 (max) |
```

File: content/load-balancing/understand-basics/limitations.md [hunk 6]
```diff
@@ -0,0 +1,13 @@
...
+| Monitors | N/A | N/A | N/A |
```
Comment (kodster28) [inline content/load-balancing/understand-basics/limitations.md]:
```suggestion
| Monitors | 1.5x the number of pools | 1.5x the number of pools | 
```

File: content/load-balancing/understand-basics/limitations.md [hunk 7]
```diff
@@ -0,0 +1,13 @@
...
+| Origin pools | 2 | 1 | 250 |
```
Comment (kodster28) [inline content/load-balancing/understand-basics/limitations.md]:
```suggestion
| Origin servers | 20 | 250 |
```

File: content/load-balancing/understand-basics/limitations.md [hunk 8]
```diff
@@ -0,0 +1,13 @@
...
+| Pools | 20 | 20 | 100 |
```
Comment (kodster28) [inline content/load-balancing/understand-basics/limitations.md]:
```suggestion
| Pools | 20 | 100 |
```

**Review by kodster28 — CHANGES_REQUESTED** [review]

File: content/load-balancing/understand-basics/limitations.md [hunk 9]
```diff
@@ -0,0 +1,14 @@
...
+pcx-content-type: reference
+title: Limitations
+weight: 18
+---
+
+# Limitations
+| Name | Non-Enterprise | Enterprise | 
+| --- | --- | --- | --- |
+| Load balancers | 20 | 100 |
+| Monitor intervals | 60 (min), 3600 (max) | 10 (min), 3600 (max) |
```
Comment (ranbel) [inline content/load-balancing/understand-basics/limitations.md]:
```suggestion
| Monitor intervals | minimum 60, maximum 3,600 | minimum 10, maximum 3,600 |
```

Comment (ranbel) [reply content/load-balancing/understand-basics/limitations.md]: style guide says to add commas to large #s.
https://wiki.cfops.it/display/Prod/Numbers

**Review by ranbel — APPROVED** [review]

**Review by kodster28 — APPROVED** [review]

**Pull request merged**